### DISCUSSION TITLE
Hexo Deploy to Heroku

### DISCUSSION THREAD

**Discussion by tomizechsterson** [hexojs/hexo#5024] [Question / Answer] [answered]
Hello,

I'm getting an error when trying to run `hexo deploy` with Heroku on Linux Mint. I think I have the right repo in `_config.yml` (assuming I'm supposed to put in the "Heroku git URL" from the Settings section in Heroku).

Here's the output in the terminal:

```
INFO  Validating config
INFO  Deploying: heroku
INFO  Clearing .deploy folder...
INFO  Copying files from public folder...
FATAL {
  err: TypeError [ERR_INVALID_ARG_TYPE]: The "mode" argument must be integer. Received an instance of Object
... stack trace...
{
    code: 'ERR_INVALID_ARG_TYPE'
  }
} Something's wrong. Maybe you can find the solution here: %s https://hexo.io/docs/troubleshooting.html
```

For what it's worth, `hexo generate` gives no errors when I run that by itself. I've also tried using the App Name from Heroku for the `repo` entry in `_config.yml`, but that yielded the same result. Is there anything more I need to do for Heroku deployment to work?

**Comment by awongCM** [comment, marked as answer]
Could this similarly related to this closed issue? - https://github.com/hexojs/hexo/issues/4263

Suggested solution found here: 👉  - https://github.com/hexojs/hexo/issues/4263#issuecomment-618887291

**Comment by tomizechsterson** [comment]
No idea, and I'm not working with these tools anymore, so I suppose we can consider this issue closed.

**Reply by awongCM** [reply]
> No idea, and I'm not working with these tools anymore, so I suppose we can consider this issue closed.

Cool. Sounds good!
